### PR TITLE
Dr listing support

### PR DESCRIPTION
**Pull request by zuzmaczek**

### CODE REVIEW

File: src/main/java/com/sinch/xms/DeliveryReportFilter.java [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+  /**
+   * Limits results to batches sent from the given addresses. If empty then all origins are
+   * included.
+   *
+   * @return a non-null set of origins
+   */
+  public abstract Set<DeliveryStatus> statuses();
+
+  /**
+   * Limits results to batches having <em>any</em> the given tags.
```
Comment (GSvensk) [inline src/main/java/com/sinch/xms/DeliveryReportFilter.java:92]: incorrect java doc?

**Review by GSvensk — CHANGES_REQUESTED** [review]
LGTM otherwise :+1:

File: src/main/java/com/sinch/xms/DeliveryReportFilter.java (hunk 1 above)
Comment (zuzmaczek) [reply src/main/java/com/sinch/xms/DeliveryReportFilter.java]: please rereview :)

**Review by thomasf147 — APPROVED** [review]

**Comment by thomasf147** [comment]
Perhaps increase the version number to 0.8.0

**Comment by zuzmaczek** [comment]
> Perhaps increase the version number to 1.1.0

Maybe we should create PR for changing version to 1.0.0 and start doing proper changelogs? 
Anyone knows how that works? There are some old ones in some unfamiliar format

**Pull request merged**